.github/workflows/release.yml

```diff
@@ -21,26 +21,29 @@ jobs:
       fail-fast: false
       matrix:
         platform:
-          - { target: aarch64-apple-darwin, os: macos-14 }
-          - { target: x86_64-apple-darwin, os: macos-13 }
-          - { target: x86_64-unknown-linux-gnu, os: ubuntu-latest }
-          - { target: x86_64-pc-windows-msvc, os: windows-latest }
+          - { target: libmozjs-aarch64-apple-darwin.tar.gz, os: macos-14 }
+          - { target: libmozjs-x86_64-apple-darwin.tar.gz, os: macos-13 }
+          - { target: libmozjs-x86_64-unknown-linux-gnu.tar.gz, os: ubuntu-latest }
+          - { target: libmozjs-x86_64-pc-windows-msvc.tar.gz, os: windows-latest }
```

Comment:
I would just keep it like it was and then just use `MOZJS_ARCHIVE: libmozjs-${{ matrix.platform.target }}.tar.gz` instead of `MOZJS_ARCHIVE: ${{ matrix.platform.target }}`.